spec/views/post/index.html.erb_spec.rb

```diff
@@ -15,7 +15,7 @@
       visit user_posts_path(subject)
 
       expect(page).to have_xpath("//img[@src='https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRJHYna2rR4LtBNlA3gUSGxsmaiYY4PyxQEcA&usqp=CAU']")
-      expect(page).to have_content('Fuad')
+      expect(page).to have_content('Joe')
       expect(page).to have_content('Number of posts: 3')
       expect(page).to have_content('Hello 1')
       expect(page).to have_content('This is my post')
```

Comment:
- [ ] Good job on writing these test cases, but please kindly include a test that asserts you can see a section for pagination if there are more posts than fit on the view